src/app/modules/item/components/item-dependencies/item-dependencies.component.ts

```diff
@@ -18,15 +23,16 @@ export class ItemDependenciesComponent implements OnChanges, OnDestroy {
   @ViewChild('op') op?: OverlayPanel;
   @ViewChildren('contentRef') contentRef?: QueryList<ElementRef<HTMLElement>>;
 
-  private readonly itemId$ = new ReplaySubject<string>(1);
+  readonly itemId$ = new ReplaySubject<string>(1);
```

Comment:
I think this subject can stay private, can you check?

Reply:
Fixed